client/my-sites/promote-post-i2/components/search-bar/style.scss

```diff
@@ -145,6 +145,9 @@ $font-sf-pro-display: "SF Pro Display", $sans;
 	}
 
 	body.is-section-promote-post-i2:not(.is-sidebar-collapsed) .main.promote-post-i2 {
+		.promote-post-i2__search-bar-wrapper .segmented-control {
```

Comment:
Can we move these types to a place where doesn't depend on the `.is-sidebar-collapsed` class?
The problem here is that these changes are not working if you refresh the page while having the sidebar collapsed.

Probably a good place would be where the class is defined a t the beginning of this file.

Reply:
Will do it :)